dts/arm/syna/sr100_b0.dtsi

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright (c) 2025, Synaptics, Inc. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <mem.h>
+#include <freq.h>
+#include <arm/armv8.1-m.dtsi>
+#include <zephyr/dt-bindings/gpio/gpio.h>
+#include <zephyr/dt-bindings/i2c/i2c.h>
+#include <zephyr/dt-bindings/clock/syna_clock.h>
+
+#define DT_FLASH_SIZE		DT_SIZE_M(4)
+#define DT_SRAM_SIZE		DT_SIZE_M(4)
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu0: cpu@0 {
+			compatible = "arm,cortex-m55";
+			device_type = "cpu";
+			reg = <0>;
+		};
+
+		cpu1: cpu@1 {
+			compatible = "arm,cortex-m4";
+			device_type = "cpu";
+			reg = <1>;
+		};
+	};
+
+	sram0: memory@33ea3000 {
+		compatible = "mmio-sram";
+		reg = <0x33ea3000 DT_SIZE_K(128)>;
+	};
+
+	flash0: memory@10000000 {
+		compatible = "mmio-sram";
+		reg = <0x10000000 DT_SIZE_K(128)>;
+	};
+
+	soc {
+		gcr: clock-controller@50330600 { /* GLOBAL */
+			reg = <0x50330600 0xa00>;
+			compatible = "syna,gcr";
+			#clock-cells = <1>;
+			status = "okay";
+		};
+
+		gcr_lp: clock-controller@b5007000 { /* M4 */
+			reg = <0xb5007000 0x100>;
+			compatible = "syna,gcr";
+			#clock-cells = <1>;
+			status = "okay";
+		};
+
+		ns16550_uart0: uart@50317000 {
+			compatible = "ns16550","snps,dw-apb-uart";
```

Comment:
missing space after , (the one between both compatibles, not inside the compatible) @kylebonnici